CRM/Pledge/BAO/PledgeBlock.php

```diff
@@ -310,8 +310,8 @@ public static function buildPledgeBlock($form) {
           switch ($field) {
             case 'contribution_date':
               $form->addDate('start_date', ts('First installment payment'));
-              $paymentDate = $value = date('d/m/Y');
-              list($defaults['start_date'], $defaults['start_date_time']) = CRM_Utils_Date::setDateDefaults($value);
+              $paymentDate = $value = date('m/d/Y');
```

Comment:
This is also the value that will be saved into database. I have already taken CiviCRM's date formats into account in the next step. https://github.com/JMAConsulting/civicrm-core/blob/e9f772df5f18cc584a3db22b704419c280bf50ff/CRM/Pledge/BAO/PledgeBlock.php#L314